tuplex/core/src/ee/local/LocalBackend.cc

```diff
@@ -693,6 +694,31 @@ namespace tuplex {
             }
         }
 
+        // assign the order for all tasks
+        for(size_t i = 0; i < tasks.size(); ++i) {
+            tasks[i]->setOrder(i);
+        }
+
+        if (tstage->hasOutputLimit()) {
+            if (tstage->outputTopLimit() > 0 && tstage->outputBottomLimit() > 0) {
```

Comment:
add comment: 3 cases:

both top and bottom limit
only top limit
only bottom limit

=> Note: Could maybe add head and tail operation to Python API?